packages/kbn-legacy-logging/src/log_interceptor.ts

```diff
@@ -55,19 +57,24 @@ function downgradeIfErrorType(errorType: string, event: AnyEvent) {
 }
 
 function downgradeIfErrorMessage(match: RegExp | string, event: AnyEvent) {
+  let tagLevel: string = 'debug';
   const isClientError = doTagsMatch(event, ['connection', 'client', 'error']);
   const errorMessage = get(event, 'error.message');
   const matchesErrorMessage = isClientError && doesMessageMatch(errorMessage, match);
 
   if (!matchesErrorMessage) {
     return null;
   }
-
+  // log in debug if OPENSSL_GET_RECORD_REGEX; log at info if OPENSSL_READ_RECORD_REGEX
```

Comment:
I think it'd be useful to add a comment about what an `OPENSSL_READ_RECORD_REGEX` match means (we can keep the link to the issue, but I think summarising in comments will save someone from clicking and reading through that)